ctapipe/reco/shower_max.py

```diff
@@ -36,11 +36,11 @@ def __init__(self, filename, col_altitude=0, col_thickness=2):
 
         self.atmosphere = Histogram(axisNames=["altitude"])
         self.atmosphere.hist = thickness * u.g * u.cm ** -2
-        self.atmosphere.bin_lower_edges = [np.array(altitude) * u.km]
+        self.atmosphere._binLowerEdges = [np.array(altitude) * u.km]
 
```

Comment:
I have another pull request pending that nobody wants to merge dealing with the renaming of this variable.
It has been suggested to not use `_binLowerEdges` but `_bin_lower_edges` which needs to be changed in the histogram class...